org.flatpak.easydict-gtk.yml

```
@@ -0,0 +1,46 @@
+app-id: org.flatpak.easydict-gtk
+runtime: org.gnome.Platform
+runtime-version: '3.38'
```

this runtime is drepecated.
```suggestion
runtime-version: '41'
```

---

OK, I will fix that.